client/js/components/payments.js

```diff
@@ -133,7 +123,11 @@ var Settings = React.createClass({
         <Paper className="dashboard-subcontent" zDepth={2}>
           <div className="dashboard-subheading">Update Rate</div>
             <div className="dashboard-subheading-content">
-              <TextField ref="newRate" className="new-rate" hintText="$" onKeyUp={this.onSubmitRateHandler} floatingLabelText={this.getRate()} />
+              <div className="rate-calculation">
+                <p className="rate-current-rate">Your current rate is: {dollarString.fromCents(this.props.settings.rate)}.</p>
+                <p className="rate-you-receive">We'll charge senders {dollarString.fromCents(this.props.settings.rate)} per email, and you'll receive {dollarString.fromCents(this.props.settings.rate * 0.7)} per email.</p>
```

Comment:
should be:

`Senders will be charged {dollarString.fromCents(Math.ceil(this.props.settings.rate * 1.029) + 30)} per email`